final_prep/mod_zero_hero.rb

```diff
@@ -47,38 +71,72 @@
 # assess_situation(99, announcement, excuse) > Should print - 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.'
 #assess_situation(21, announcement, excuse) > should print - 'Never fear, the Courageous Curly Bracket is here!'
 #assess_situation(3, announcement, excuse) > should print - "Meh. Hard pass."
-
+puts assess_situation(99, announcement, excuse)
+puts assess_situation(21, announcement, excuse)
+puts assess_situation(3, announcement, excuse)
 # Declare a new variable - scary_monster - assigned to an hash with the following key/values
+
 #   - name (string)
 #   - smell (string)
 #   - weight (integer)
 #   - citiesDestroyed (array)
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
-
+scary_monster = {
+ "name" => "Phil Mickelson",
+ "smell" => "Coffee and Sarcasm",
+ "weight" => 190,
+ "citiesDestroyed" => ["Scottsdale", "Augusta", "Del Monte Forest"],
+ "luckyNumbers" => [1, 88, 3],
+ "address" => {number:"112", street:"Calves Ave.", state: "CO", zip: "80033"}
 
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
+class SuperHero
+  attr_accessor :name, :super_power, :age
+
+  arch_nemesis= "The Syntax Error"
+  power_level = 100
+  energy_level = 50
+
+  def say_name
+    puts "#{name} Will save us!"
+  end
+
+  def maximize_energy
+    energy_level = 1000
+  end
+
+  def gain_power(num)
+    @num = num
```

Comment:
This method does need a parameter, so good job there. But you need a reassignment of one of your attributes. Think about how to assign that attribute to itself plus the `num` passed through as a parameter. Right now, there is no reassignment.